### PR TITLE
Fix part of #116

### PR DESCRIPTION
**Pull request by SanjayDevTech**
## Explanation
I have implemented group chat community feature.
Used Realtime database to fetch chats and used Firebase-UI to display
Since this PR won't contain full fledged chat feature but it covers the basic functionality of chat.
This PR Fixes part of #116

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".

### CODE REVIEW

**Comment by SanjayDevTech** [comment]
image

This is the structure of the msg in the database.
`content` => Actual text content
`createdTime` => The timestamp at which the msg was sent
`msgId` => The key of the msg object
`senderId` => UId of the sender
`senderName` => Name of the sender (This can be retrieved from getDisplayName() method of FirebaseUser class.. This works only for Google or any other social login... It won't work for Email password validation)

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Remove build.gradle and gradle wrapper chnages

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```
Comment (riturajjain2000) [inline gradle/wrapper/gradle-wrapper.properties:6]: here

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.2"
```
Comment (riturajjain2000) [inline build.gradle:11]: here

**Comment by SanjayDevTech** [comment]
image

File: build.gradle (hunk 2 above)
Comment (SanjayDevTech) [reply build.gradle]: : ) Done

**Pull request merged**